### ISSUE TITLE
A11y_Visual Studio Code_Footer Section_AI4W: Ensures ARIA attributes are allowed for an element's role (iframe[frameborder="\30 "])

### ISSUE THREAD

**Issue by NiharikaTipirishetty** [microsoft/vscode-docs#4525]
GitHubTags:#A11y_Visual Studio Code#A11yAuto#A11ySev2#A11yTCS#A11yMAS#Chrome#DesktopWeb#Win10#Visual Studio Code Services#WCAG4.1.2#aria-allowed-attr#VisualStudioCode-Web-Apr2021

### Environment details:
Application Name: Visual Studio Code
URL:[Visual Studio Code](https://code.visualstudio.com/docs/setup/linux)
Windows Version: Version 20H2 (OS Build 19042.928)
Browser: Chrome Version 90.0.4430.85 (Official Build) (64-bit)

### Issue: 
Ensures ARIA attributes are allowed for an element's role (aria-allowed-attr - https://accessibilityinsights.io/info-examples/web/aria-allowed-attr)

### Target application: 
Running Visual Studio Code on Linux - https://code.visualstudio.com/docs/setup/linux#main-content

### Element path:
iframe[frameborder="\30 "]

### Snippet: 
<iframe src="/assets/github-button.html?user=Microsoft&amp;repo=vscode&amp;type=star&amp;count=true" frameborder="0" scrolling="0" width="130px" height="20px" title="GitHub follow button" aria-label="GitHub follow button"></iframe>

### How to fix: 
Fix all of the following:
  ARIA attribute cannot be used: aria-label

Environment: Chrome version 90.0.4430.85

====

This accessibility issue was found using Accessibility Insights for Web 2.26.0 (axe-core 4.2.0), a tool that helps find and fix accessibility issues. Get more information & download this tool at http://aka.ms/AccessibilityInsights.
### Attachment:
![A11y_Visual Studio Code_Footer Section_AI4W](https://user-images.githubusercontent.com/83570638/117129486-5bc49980-adbc-11eb-9062-fe918a39fa84.png)

**Comment by sana-ajani** [comment]
Hi there, based on this: https://www.w3.org/TR/wai-aria-1.1/#button and https://www.tpgi.com/short-note-on-aria-label-aria-labelledby-and-aria-describedby/, it looks like we can use `aria-label` for button and iframes. Thus I think this issue is incorrect.

**Comment by sana-ajani** [comment]
I was mistaken. After doing more research (https://developer.mozilla.org/en-US/docs/Web/HTML/Element/iframe), I see that since the role is generic, there is no implicit ARIA role. I will remove that!

**Comment by rzhao271** [comment]
Confirmed fix in PPE.

**Comment by kalpanah1** [comment]
#Closed;
Verified the issue in the below environment. The issue is fixed please find the attachment for reference.
URL: https://code.visualstudio.com/
![Bug_4525_Fixed](https://user-images.githubusercontent.com/94966876/218005789-e9cc5656-514a-4674-8625-74960f6572b7.png)
